plugins/_getdir/utils.js

```diff
@@ -0,0 +1,22 @@
+function filterDir(e) {
+	if (e.key === "Escape") {
+		e.target.value = "";
+	}
+	let keyword = e.currentTarget.value.toUpperCase();
+	let dirs = document.querySelector(".rmenuobj").querySelectorAll("div");
+	for (let i = 0; i < dirs.length; i++) {
+		if (dirs[i]) {
+			let txtValue = dirs[i].textContent || dirs[i].innerText;
+			dirs[i].style.display = txtValue.toUpperCase().indexOf(keyword) > -1 ? "" : "none";
+		}
+	}
+}
+
+document.getElementById("dir-search-bar").addEventListener("keyup", filterDir);
+document.addEventListener("readystatechange", () => {
+	if (document.readyState === "complete") {
+		setTimeout(() => {
+			document.getElementById("dir-search-bar").focus()
+		}, 500);
+	}
+});
```

Comment:
Added this event listener when the iframe is in ready state. I had to use the `setTimeout` function, otherwise JavaScript wouldn't help us autofocus on the text input.

Reply:
`setTimeout` can be set to `0`. Please test to confirm this works for you. Also please let a comment that `setTimeout` is required to queue the focus last in sequence, so we don't accidently remove it.

Reply:
Looks like it's a browser related issue. I've tried Firefox and Chrome, both the latest version. Even a plain `.focus()` without `setTimeout()` would work perfectly on Chrome, but not on Firefox. 

I've tried `0` and something below `100`. The autofocus functionality just worked occasionally. A timeout value of `100` seems to be a fair workaround. Our users could barely notice the time gap.

Reply:
OK, that's the way to go. I'll take some time looking into that part. It will probably need a few more days, because I've been busy with work lately.